src/lsif.ts

```diff
@@ -2738,7 +2738,7 @@ class Visitor implements ResolverContext {
     let monikerIdentifer: string | undefined
     if (tss.isSourceFile(symbol) && monikerPath !== undefined) {
       monikerIdentifer = tss.createMonikerIdentifier(monikerPath, undefined)
-    } else if (exportPath !== undefined) {
+    } else if (exportPath !== undefined && exportPath !== '') {

```

Comment:
I guess this is probably one of those rare cases where it would (have )be(en) OK to rely on implicit truthiness.

But LGTM.